src/lesson7/Circle.java

```diff
@@ -0,0 +1,25 @@
+package lesson7;
+
+public class Circle extends Shape {
+    private double pi = 3.14;
+    private double r;
```

Comment:
PI це костанта яка повинна належати класу
```suggestion
    private static final double PI = 3.14;
    private double r;
```